Example/PaymentSheet Example/PaymentSheet Example/SavedPaymentMethodSheetTestPlayground.swift

```diff
@@ -141,28 +160,30 @@ class SavedPaymentMethodSheetTestPlayground: UIViewController {
     }
 
     func savedPaymentMethodSheetConfiguration(customerId: String, ephemeralKey: String) -> SavedPaymentMethodsSheet.Configuration {
-        let customerContext = _stpspmsbeta_STPCustomerContext(customerId: customerId, ephemeralKeySecret: ephemeralKey)
-        self.customerContext = customerContext
-        var configuration = SavedPaymentMethodsSheet.Configuration(customerContext: customerContext,
-                                                                   applePayEnabled: applePayEnabled())
-        configuration.createSetupIntentHandler = setupIntentHandler(customerId: customerId)
+        let customerAdapter: StripeCustomerAdapter
+        switch paymentMethodMode {
+        case .setupIntent:
+            customerAdapter = StripeCustomerAdapter(customerEphemeralKeyProvider: {
+                // This should be a block that fetches this from your server
+                .init(customerId: customerId, ephemeralKeySecret: ephemeralKey)
```

Comment:
To ensure nobody removes `async` from the signature of StripeCustomerAdapter's `customerEphemeralKeyProvider` parameter, this line should probably be:

```
return await self.someFunc(customerId: customerId, ephemeralKey: ephemeralKey)

.... then....

    func someFunction(customerId: String, ephemeralKey: String) async -> CustomerEphemeralKey {
         // This should be a block that fetches this from your server
        return .init(customerId: customerId, ephemeralKeySecret: ephemeralKey)
    }
```
This way, if anyone removes the `async` keyword from the parameter, then this will fail to compile.

Reply:
I think it's okay if a user doesn't treat this as async: They just won't get the benefits of customer ephemeral key auto-refreshing. Our example backend isn't currently set up for it, as it creates a new customer ephemeral key each time and doesn't support fetching an ephemeral key for an arbitrary customer. 🙈

Reply:
I think it's also okay if the user doesn't treat it as async -- i guess my point is more of a nit that if someone accidentally removes the `async` keyword on the init of CustomerAdapter -- e.g.:
from:
```
    public init(customerEphemeralKeyProvider: @escaping () async throws -> CustomerEphemeralKey,
```
to:
```
    public init(customerEphemeralKeyProvider: @escaping () throws -> CustomerEphemeralKey,
```
Our code will continue to compile, but I think that compilation should fail, because there will be some customers who do implement an async keyProvider.  By changing our code to pass in fake "async" key provider, we will have a compile time check that nobody removes the `async` keyword.